package.json

```diff
@@ -111,7 +109,6 @@
     "jsdom": "^16.4.0",
     "jsdom-global": "^3.0.2",
     "mocha": "^8.1.3",
-    "node-notifier": "^8.0.0",
     "normalize.css": "^8.0.1",
     "nswatch": "^0.2.0",
```

Comment:
We might not need this if we got rid of the legacy watch tasks.

Reply:
Removed.